on_guild_join.py

```diff
@@ -0,0 +1,25 @@
+import discord
+
+async def intialize_order_channels(guild, client, confirmed_channel_name, order_channel_name): 
+    confirmed_order_channel = None
+    order_channel = None
+    for chnl in client.get_all_channels():
+        if chnl.name == confirmed_channel_name:
+            confirmed_order_channel = chnl
+            continue
+
+        if chnl.name == order_channel_name:
+            order_channel = chnl
+            order_channel.purge(check=lambda m: m.author == client.user)
+            continue
+
+    if confirmed_order_channel == None:
+        permissions = {
+             guild.default_role: discord.PermissionOverwrite(send_messages=False)
```

Comment:
The default_role attribute sets for the guild's @everyone role - does this override roles higher than it in the hierarchy or does it limit everyone except the bot?